app/questionnaire/placeholder_transforms.py

```diff
@@ -183,3 +183,8 @@ def first_non_empty_item(self, items):
             return item
 
         return ""
+
+    def format_suggestions_url(self, url, list_json):
+        if self.language == "en":
```

Comment:
Don't we also want to concatenate when it's "cy"?

Reply:
No, because "cy" is not used in this transform, it's a different placeholder altogether. If I started transforming "cy" and "en" I would need to know which "en" I'm transforming from a region code.

Reply:
Then I think this transform should be renamed `format_nisra_suggestions_url` or something similar as at the moment ithe naming implies it can be used against any url.

Reply:
I had it as `format_nisra_suggestions_url` at first but changed it to more universal name after and forgot to change it back when I realised that region code for Eng "en" will be needed🤦‍♂️.